ebos/eclproblem.hh

```diff
@@ -337,7 +337,8 @@ SET_TYPE_PROP(EclBaseProblem, NewtonMethod, Opm::EclNewtonMethod<TypeTag>);
 SET_INT_PROP(EclBaseProblem, RestartWritingInterval, 0xffffff); // disable
 
 // Drift compensation is an experimental feature, i.e., systematic errors in the
-// conservation quantities are only compensated for if experimental mode is enabled.
+// conservation quantities are only compensated for
+// as default if experimental mode is enabled.
```

Comment:
I think this comment is wrong: EnableExperiments and EclEnableDriftCompensation seem independent now, to me?

Reply:
With this PR Flow defaults EclEnableDriftCompensation to false while Ebos to true. (Since EnableExperiments is true in Ebos) So not completely independent.

Reply:
I did not notice that the default was still set as before, so you are of course correct.